test/ecpair.js

```diff
@@ -44,31 +44,6 @@ describe('ECPair', function () {
       assert.strictEqual(keyPair.network, NETWORKS.testnet)
     })
 
-    it('throws if compressed option is not a bool', function () {
-      assert.throws(function () {
-        new ECPair(null, null, {
-          compressed: 2
-        }, /Expected Boolean, got 2/)
-      })
-    })
-
-    it('throws if public and private key given', function () {
-      var qBuffer = new Buffer('0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798', 'hex')
-      var Q = ecurve.Point.decodeFrom(curve, qBuffer)
-
-      assert.throws(function () {
-        new ECPair(BigInteger.ONE, Q)
-      }, /Unexpected publicKey parameter/)
-    })
-
-    it('throws if network is missing pubKeyHash constants', function () {
-      assert.throws(function () {
-        new ECPair(null, null, {
-          network: {}
-        }, /Unknown pubKeyHash constants for network/)
```

Comment:
Funny enough,  this wasn't even being tested.  The regex is in the constructor call by mistake.